### PR TITLE
Heart-Shaped Box

### PR DESCRIPTION
**Pull request by Evilpriest4**
Doing the first Event Dungeon Finder of the day currently won't award any Emblem of Frost and will aswell count as the first RDF of the day, thus making impossible to get the first two EoFs of the day (with RDFs).

**Changes proposed:**
Adding 2x Emblem of Frost inside the Heart-Shaped Box.

**Target branch(es):** master

**Tests performed:** Tested in game.

### CODE REVIEW

**Comment by talamortis** [comment]
please fix travis, i have told u whats wrong with him then ill test 👍

File: data/sql/updates/pending_db_world/rev_1518477962308987200.sql [hunk 1]
```diff
@@ -1,7 +1,7 @@
...
 
---Evilpriest #0003
---Added EoF from Heart-Shaped Box
+#--Evilpriest #0003
+#--Added EoF from Heart-Shaped Box
 
 INSERT INTO `item_loot_template` (`entry`,`item`,`ChanceOrQuestChance`,`lootmode`,`groupid`,`mincountOrRef`,`maxcount`) VALUES
-('54537','49426','100','1','0','2','2')
\ No newline at end of file
+('54537','49426','100','1','0','2','2')
```
Comment (Nefertumm) [inline data/sql/updates/pending_db_world/rev_1518477962308987200.sql]: Add a ; at the end of the line.

File: data/sql/updates/pending_db_world/rev_1518477962308987200.sql [hunk 2]
```diff
@@ -0,0 +1,7 @@
+INSERT INTO version_db_world (`sql_rev`) VALUES ('1518477962308987200');
+
+#--Evilpriest #0003
+#--Added EoF from Heart-Shaped Box
+
+INSERT INTO `item_loot_template` (`entry`,`item`,`ChanceOrQuestChance`,`lootmode`,`groupid`,`mincountOrRef`,`maxcount`) VALUES
+('54537','49426','100','1','0','2','2')
```
Comment (ShinDarth) [inline data/sql/updates/pending_db_world/rev_1518477962308987200.sql]: you need to `DELETE` before insert, the SQL queries should be re-executable when possible.

**Comment by sucofog** [comment]
Is this blizzlike?

https://wotlk.evowow.com/?item=54537#contains

**Comment by Evilpriest4** [comment]
@sucofog These are the references I could find:
**Inside the core:**
Similar occurences of this first Event Dungeon Finder of the day can be found in those IDs: 
-54536 (Satchel of Chilled Goods) (Midsummer)
-54535 (Keg-Shaped Treasure Chest) (Brewfest)
-54516 (Loot-Filled Pumpkin) (Hallow's End)
In every of these there are already 2x EoFs set as loot by DB.

**Elsewhere:**
[https://us.battle.net/forums/en/wow/topic/2029914992](url) In this post the author references to the Heart-Shaped Box loot not being aligned with Cataclysm currencies. He also specifies that it used to drop EoFs.
[http://www.wowhead.com/love-is-in-the-air#comments](url) Here the user 'Nolimit' points out that there is an issue with him not dropping any items from the Box and that he has received a response from a GM, and he states "no loot from box after doing brothers(said that's due to emblems>JP switch)".

**Comment by Knindzagxg** [comment]
There's change to drop Emblems, that's true.

**Comment by Knindzagxg** [comment]
There should be no any Emblem reward in RDF screen and not even in the box.
When you queue up, 2 rewards should be up: Box and currency for event that is into progress or extra gold, depends on the event.
Boss it self, should award Emblems on kill, like in Dungeons.

**Comment by FrancescoBorzi** [comment]
Looking for @Evilpriest4  or someone else to completed the work that has been previously started here.

**Comment by Pondaveia** [comment]
So, can anyone confirm:
1. RDF doesn't award the emblems but the last boss?
2. The box doesn't contain emblems?

**Comment by talamortis** [comment]
Since the DB has changed the sql is incorrect, I will fix this either tonight or tomorrow.

**Comment by Kitzunu** [comment]
```sql
DELETE FROM `item_loot_template` WHERE `Entry` = 54537 AND `Item` = 49426;
INSERT INTO `item_loot_template` (`Entry`, `Item`, `Reference`, `Chance`, `QuestRequired`, `LootMode`, `GroupId`, `MinCount`, `MaxCount`, `Comment`) VALUES 
(54537, 49426, 0, 100, 0, 1, 0, 2, 2, '');
```
Fixed :)

File: data/sql/updates/pending_db_world/rev_1518477962308987200.sql [hunk 3]
```diff
@@ -0,0 +1,5 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1518477962308987200');
+
+DELETE FROM `item_loot_template` WHERE (`entry` = 54537 AND `item` = 49426);
+INSERT INTO `item_loot_template` (`entry`,`item`,`ChanceOrQuestChance`,`lootmode`,`groupid`,`mincountOrRef`,`maxcount`) VALUES
+('54537','49426','100','1','0','2','2');
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1518477962308987200.sql:5]:
```suggestion
DELETE FROM `item_loot_template` WHERE `Entry` = 54537 AND `Item` = 49426;
INSERT INTO `item_loot_template` (`Entry`, `Item`, `Reference`, `Chance`, `QuestRequired`, `LootMode`, `GroupId`, `MinCount`, `MaxCount`, `Comment`) VALUES 
(54537, 49426, 0, 100, 0, 1, 0, 2, 2, '');
```
Fixed .. :)

**Comment by LukasVolgger** [comment]
Updated. Thx @Kitzunu!

**Comment by Kitzunu** [comment]
how to test:
1. .add 54537
2. open

**Comment by BarbzYHOOL** [comment]
Worked fine, it dropped emblems + usual items

**Comment by BarbzYHOOL** [comment]
thank you guys, merged before we forget

**Pull request merged**